src/commonMain/kotlin/io/github/koalaplot/core/xygraph/XYGraph.kt

```diff
@@ -627,6 +668,8 @@ public interface XYGraphScope<X, Y> : HoverableElementAreaScope {
     public val yAxisModel: AxisModel<Y>
     public val xAxisState: AxisState
     public val yAxisState: AxisState
+    public var mouseToChartOffset: IntOffset
+    public var graphSize: IntSize
```

Comment:
This is leaking screen coordinate information to the user which should be avoided. I think you are trying to make it so the user is reported the mouse position in real time. The problem with this approach is that the user gets the coordinates in screen space and not axis coordinate space, and so cannot do anything relative to the data that is being plotted without inverting the transform being performed by the ```AxisModel```s, i.e., ```computeOffset```.